nikola/nikola.py

```diff
@@ -1984,6 +1984,20 @@ def create_hierarchy(cat_hierarchy, parent=None):
         # Next, flatten the hierarchy
         self.category_hierarchy = utils.flatten_tree_structure(root_list)
 
+    def sort_posts(self, posts, lang=None):
```

Comment:
I first wanted to do it inplace, but that's kind of annoying with `natsort` (you need to use `natsort.natsort_keygen`, but I'm not sure whether that leads to the same order as `natsort.natsorted`). But if you insist I can change it back.